components/x-live-blog-post/readme.md

```diff
@@ -52,3 +52,5 @@ Feature             | Type   | Notes
 `publishedTimestamp`| String | Deprecated - ISO timestamp of publish date
 `articleUrl`        | String | Url of the main article that includes this post
 `showShareButtons`  | Bool   | default: `false` - Shows social media share buttons when `true`
+`backToTop`  		| Function | String   | Shows the back to top link at the bottom of posts and manages navigating to `selected top` with a javascript function or a hashed href. Please call event.preventDefault() at the top level if this prop is a function. If prop is a string, add the string as the `id` to the element that represents the top.
```

Comment:
**Minor improvements:** Looks like the indentation is slight off here also some small improvements to the setence structure to help keep it clear.

```suggestion
`backToTop`                | String | Function   | Shows the back to top link at the bottom of posts and manages navigating to `selected top` with a javascript function or a hashed href (string). If this prop is a string it will rely on standard browser behaviour to navigate to the element `id` provided that represents the top. If this prop is a function then that function should control the experience of navigating/scrolling to the top position. When using a function please call event.preventDefault() at the top level.
```